internal/ethapi/api.go

```diff
@@ -1670,6 +1673,75 @@ func (s *PublicTransactionPoolAPI) GetTransactionReceipt(ctx context.Context, ha
 	return fields, nil
 }
 
+// GetReceiptProof returns a proof which is the receipt-tree(mpt) path for the receipt corresponding to the specified block to verify the validity of the receipt
+func (s *PublicTransactionPoolAPI) GetReceiptProof(ctx context.Context, txHash common.Hash) (map[string]interface{}, error) {
+	_, blockHash, blockNumber, index, err := s.b.GetTransaction(ctx, txHash)
+	if err != nil {
+		return nil, err
+	}
+	receipts, err := s.b.GetReceipts(ctx, blockHash)
+	if err != nil {
+		return nil, err
+	}
+	if len(receipts) <= int(index) {
+		return nil, fmt.Errorf("index invalid")
+	}
+	receipt := receipts[index]
+	//1. rlp encode receipt value
```

Comment:
```suggestion
	// 1. rlp encode receipt value
```

Need a space between "//" and the following text. (Apply below)